kernel/Dockerfile

```diff
@@ -98,8 +98,16 @@ RUN make defconfig && \
     cp System.map /out && \
     ([ -n "${DEBUG}" ] && cp vmlinux /out || true)
 
+# WireGuard
+RUN TAG="0.0.$(curl -sSL https://git.zx2c4.com/WireGuard/info/refs/ | sed -n 's/.*refs\/tags\/0\.0\.\([0-9]\+\)$/\1/p' | sort -nr | head -n 1)" && \
```

Comment:
For consistency, I would prefer to use the pattern we use at different places like [here](https://github.com/linuxkit/linuxkit/blob/master/pkg/init-lcow/Dockerfile#L11). Basically define and environment variable for `WIREGUARD_REPO` and `WIREGUARD_COMMIT`.